lib/tasks/whitehall_importer.rb

```diff
@@ -50,10 +51,13 @@ def log_error(error)
 
     def create_users(users)
       users.each do |user|
-        next if User.exists?(uid: user["uid"])
-
         user_keys = %w[uid name email organisation_slug organisation_content_id]
-        User.create!(user.slice(*user_keys).merge("permissions" => []))
+        content_publisher_user = User.create_with(user.slice(*user_keys).merge("permissions" => [])).find_or_create_by!(uid: user["uid"])
+        user_map.push(
+          "uid" => user["uid"],
+          "whitehall_user_id" => user["id"],
+          "content_publisher_user_id" => content_publisher_user["id"],
+        )
```

Comment:
I wonder if it'd be easier to traverse this as a hash of whitehall user_ids to content publisher user ids? I assume we don't really use the uid part after this method. That might give some clues to adjust naming too.

Reply:
Indeed, I've updated the PR.  I was planning to use the UIDs at some other point, but turns out they weren't needed for anything.